File: src/main/java/com/example/gamezcms/DTO/MenuGenreDTO.java
package com.example.gamezcms.DTO;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.util.List;

@Data
@AllArgsConstructor
public class MenuGenreDTO {
    private Long id;
    private String name;
    private List<MenuArticleDTO> articlesList;
}

File: src/webapp/src/app/model/menu-genre-dto.ts
import { ImmutableObject } from "@angular/cdk/overlay/typings/overlay-ref";
import { KeyValue } from "@angular/common";
import { MenuArticleDTO } from "./menu-article-dto";

export class MenuGenreDTO {
    constructor(
        public id: number,
        public name: string,
        public articlesList: Array<MenuArticleDTO>) {}
}
